### ISSUE TITLE
funcall: Symbol’s value as variable is void: cl-struct-magit-section-tags

### ISSUE THREAD

**Issue by Jehops** [magit/magit#3340]
After upgrading the magit package from melpa, `magit-status` reports the error in the subject and the status is not shown [1].  `M-x magit-version` reports `Magit 20180120.435, Git 2.16.0, Emacs 25.3.1, berkeley-unix`.  The problem went away using the command from `M-x magit-emacs-Q-command RET`, so I debugged my configuration and found removing the `magit-svn` package fixed the problem.  After reinstalling `magit-svn`, `magit-status` worked as expected again.

In summary, everything is working as expected again and this issue is just a notice for you and anyone else who also has this problem.

[1] The problem may have only been for repositories with `[magit] extension = svn`.  I have since made the changes described above, so I cannot confirm this.

**Comment by kyleam** [comment]
Have you recompiled/reinstalled magit-svn?  

Re: #3336

**Comment by vermiculus** [comment]
@kyleam 

> After reinstalling `magit-svn`, `magit-status` worked as expected again.
>
> In summary, everything is working as expected again and this issue is just a notice for you and anyone else who also has this problem.

This issue can probably be closed.

**Comment by Jehops** [comment]
Oh, geez.  I did look at the issue history, but somehow missed #3336.

**Comment by kyleam** [comment]
Sorry, @Jehops, as @vermiculus points out, I should have more
carefully read your message :/

> The problem may have only been for repositories with [magit] extension = svn

I think any non-Magit code that is compiled and uses the old struct
sections would need to be recompiled.  Some of the extensions needed
other updates for 651a9ab (e.g., magit/magit-annex#17), and @tarsius
merged these changes in at around the same time, so users that upgrade
should get the non-struct magit and recompiled extension at the same
time, avoiding the issue.
